features/step_definitions/edit_delete_task_steps.rb

```diff
@@ -0,0 +1,5 @@
+Given(/^I am logged in as "([^"]*)" and password "([^"]*)"$/) do |email, arg2|
```

Comment:
Perhaps word it as "... logged in as requester ..."
And the password is not needed? It is listed as arg2, but seems not to be used?